ProjectShop/ProjectShop/urls.py

```diff
@@ -14,10 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
```

Comment:
Path is already imported

(Here we have 2 imports of path)

Reply:
?